image/roxctl/rhtap.Dockerfile

```diff
@@ -0,0 +1,53 @@
+# TODO: can we follow tags?
```

Comment:
Let's replace this TODO with pointers that we found.

Something like this:

```suggestion
# openshift-golang-builder is the only way to get more recent Go version than the official ubi8/go-toolset provides.
# See https://issues.redhat.com/browse/RHELPLAN-167618
# Using that has few known issues:
# - https://issues.redhat.com/browse/RHTAPBUGS-864 - deprecated-base-image-check behaves incorrectly.
# - https://issues.redhat.com/browse/RHTAPBUGS-865 - openshift-golang-builder is not considered to be a valid base image.
#
# Also, we can't pin image tag or digest because currently there's no mechanism to auto-update that.
# See https://issues.redhat.com/browse/STONEBLD-1823
# We're targeting a floating tag here which should be reasonably safe to do as both RHEL major 8 and Go major.minor 1.20 should provide enough stability.
```

FYI, I was looking at suggestions provided around RHTAPBUGS-865 but it was difficult so it will take more time.